### PR TITLE
Drop # from urls

### LINKED ISSUE #160: Reddit treats all Streamium URLs as the same, throws "link has already been submitted" errors.
Yemel had an idea for how to fix this.

### PR DESCRIPTION
**Pull request by yemel**
Fixes #160 

Summary:

* Improve url names like *streamium.io/s/yemel*
* The server requires some url mappings.
* Update *npm start* command.

### CODE REVIEW

**Comment by eordano** [comment]
Update package.json: add `express`

**Comment by yemel** [comment]
Updated

File: index.html [hunk 1]
```diff
@@ -1,6 +1,108 @@
-<html>
+<!DOCTYPE html>
```
Comment (maraoz) [inline index.html]: why is all this code shown as new code?

Comment (eordano) [inline index.html]: Because of not using `git mv`? It's all right I feel

**Pull request merged**